french_law/ocaml/law_source/aides_logement.ml

```diff
@@ -4812,7 +4788,7 @@ let base_mensuelle_allocations_familiales
                handle_default
                  {
                    filename =
-                     "examples/aides_logement/../base_mensuelle_allocations_familiales/bmaf.catala_fr";
+                     "./../base_mensuelle_allocations_familiales/bmaf.catala_fr";
```

Comment:
So this probably won't work. We can't have `../` in these paths, as they mess up the display of things in the website HTML-wise. That's why we made sure to call `catala` from the root directory to generate these OCaml files so that the paths all begin with `examples/`.

Reply:
Ah right! I had noticed this then forgot to check :/
I'll find a fix.